src/Moryx.AbstractionLayer.TestTools/Moryx.AbstractionLayer.TestTools.csproj

```diff
@@ -8,14 +8,9 @@
     <PackageTags>MORYX;IIoT;IoT;Abstraction;Tests</PackageTags>
   </PropertyGroup>
 
-  <ItemGroup>
-	<PackageReference Include="Microsoft.EntityFrameworkCore" />
-  </ItemGroup>
-
   <ItemGroup>
 	<ProjectReference Include="..\Moryx.Model.Sqlite\Moryx.Model.Sqlite.csproj" />
 	<ProjectReference Include="..\Moryx.AbstractionLayer\Moryx.AbstractionLayer.csproj" />
-	<ProjectReference Include="..\Moryx.TestTools.UnitTest\Moryx.TestTools.UnitTest.csproj" />
```

Comment:
why is that not needed anymore? Was that ever needed?

Reply:
It was imported in multiple projects but the automatic detection of unused imports from visual studio detected it multiple times as not use. Maybe @Toxantron can tell us what it was originally used/imported for 🤔